bucket/dos2unix.json

```
@@ -21,6 +21,16 @@
     ],
     "checkver": {
         "url": "https://waterlan.home.xs4all.nl/dos2unix.html",
-        "re": "Latest version: (?<version>[\\d.]+) \\([\\d\\-]+\\)"
+        "re": "Latest version: ([\\d.]+) \\([\\d\\-]+\\)"
```

Do you need to be so strict?

```suggestion
        "regex": "Latest version: ([\\d.]+)\\s+\\(\\d"
```
Should be enough. Or?